index.rst

```diff
@@ -35,6 +35,7 @@ The framework consists of a suite of projects, tools, and libraries which may be
    :hidden:
 
    flux-core <https://flux-framework.readthedocs.io/projects/flux-core/en/latest/index.html>
+   flux-secuirty <https://flux-framework.readthedocs.io/projects/flux-security/en/latest/index.html>
```

Comment:
typo: `flux-secuirty`

Reply:
Just rebased on #129 so this commit was dropped.